### PR TITLE
WL- 4148 Having isValidDomain in kernel Validator and use it to validate email domain 

### PR DESCRIPTION
**Pull request by surya-janani**

### CODE REVIEW

**Comment by surya-janani** [comment]
linked with https://github.com/ox-it/wl-kernel/pull/75

**Comment by surya-janani** [comment]
linked with https://github.com/ox-it/wl-site-manage/pull/51

File: user-tool/tool/src/java/org/sakaiproject/user/tool/UsersAction.java [hunk 1]
```diff
@@ -1096,7 +1097,12 @@ else if ((mode != null) && (mode.equalsIgnoreCase("edit")) && (!singleUser))
 				addAlert(state, rb.getString("useact.invemail"));	
 				return false;
 		}
-		
+		if(ServerConfigurationService.getBoolean("edituser.validate.emailDomain",true)&& !Validator.isValidDomain(email)) {
```
Comment (buckett) [inline user-tool/tool/src/java/org/sakaiproject/user/tool/UsersAction.java]: Code formatting. Missing spaces here.

**Comment by surya-janani** [comment]
Review changes and rebased

**Pull request merged**